lib/inx/extensions.py

```diff
@@ -41,7 +41,7 @@ def generate_extension_inx_files():
             continue
 
         name = extension.name()
-        template = env.get_template('%s.inx' % name)
+        template = env.get_template('%s.xml' % name)
```

Comment:
Inkscape 1.0 is greedy and will put every .inx file it can find into the menu. This isn't needed for the final build packages, but for developers we don't want to blow up the menu.